keps/sig-storage/20191117-immutable-secrets-configmaps.md

```diff
@@ -128,11 +128,16 @@ field:
 ```
 
 If set, the machinery in apiserver will reject any updates of the object
-trying to change anything different than ObjectMetadata. Note that we
-need to allow mutating ObjectMetadata to not break object lifecycle
-(e.g. to avoid deadlock if Finalizers are set).
+trying to change anything different than ObjectMetadata.
 
-Based on the value of that field Kubelet will or will not:
+Note that will NOT reject all updates of the object, as we need to allow
+e.g. for mutating ObjectMetadata (to not break object lifecycle, e.g. by
+introducing a deadlock if Finalizers are set) or to allow rotating
+certificates used for encryption at rest. We will only reject requests
+that are explicitly changing keys and/or values stored in Secrets and/or
+ConfigMaps.
```

Comment:
@derekwaynecarr - I rephrased this paragraph to make it explicit what we discussed yesterday during the SIG.
PTAL

Reply:
thanks for clarification.